src/main/resources/application.yaml

```diff
@@ -394,12 +489,117 @@ spring.cloud.azure.reference:
         - lowest-spring-boot-version: 2.7.0
           highest-spring-boot-version: 2.7.3
           version: 4.4.1
+    - artifact-id: spring-cloud-azure-starter-jdbc-mysql
+      lowest-spring-boot-version: 2.5.0
+      highest-spring-boot-version: 2.7.3
+      mappings:
+        - lowest-spring-boot-version: 2.5.0
+          highest-spring-boot-version: 2.5.14
+          version: 4.5.0-beta.1
+        - lowest-spring-boot-version: 2.6.0
+          highest-spring-boot-version: 2.6.12
+          version: 4.5.0-beta.1
+        - lowest-spring-boot-version: 2.7.0
+          highest-spring-boot-version: 2.7.3
+          version: 4.5.0-beta.1
+        - lowest-spring-boot-version: 3.0.0-RC1
+          highest-spring-boot-version: 3.0.0-RC1
+          version: 6.0.0-beta.3
+    - artifact-id: spring-cloud-azure-starter-jdbc-postgresql
+      lowest-spring-boot-version: 2.5.0
+      highest-spring-boot-version: 2.7.3
+      mappings:
+        - lowest-spring-boot-version: 2.5.0
+          highest-spring-boot-version: 2.5.14
+          version: 4.5.0-beta.1
+        - lowest-spring-boot-version: 2.6.0
+          highest-spring-boot-version: 2.6.12
+          version: 4.5.0-beta.1
+        - lowest-spring-boot-version: 2.7.0
+          highest-spring-boot-version: 2.7.3
+          version: 4.5.0-beta.1
+        - lowest-spring-boot-version: 3.0.0-RC1
+          highest-spring-boot-version: 3.0.0-RC1
+          version: 6.0.0-beta.3
   versions:
     - artifact-id: spring-cloud-azure-dependencies
       ga-version: 4.4.1
-      preview-version: 4.0.0-beta.3
+      preview-version: 6.0.0-beta.3
     - artifact-id: spring-cloud-azure-starter-keyvault-certificates
-      has-preview-version: false
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
     - artifact-id: azure-security-keyvault-jca
       ga-version: 2.7.0
-      preview-version: 1.0.0-beta.7
+      has-preview-version: false
+    - artifact-id: spring-cloud-azure-starter-active-directory
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
+    - artifact-id: spring-cloud-azure-starter-active-directory-b2c
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
+    - artifact-id: spring-cloud-azure-starter-appconfiguration
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
+    - artifact-id: spring-cloud-azure-starter-data-cosmos
+      ga-version: 4.4.1
+      has-preview-version: false
+    - artifact-id: spring-cloud-azure-starter-cosmos
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
+    - artifact-id: spring-cloud-azure-starter-keyvault
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
+    - artifact-id: spring-cloud-azure-starter-keyvault-secrets
+      ga-version: 4.4.1
+      preview-version: 6.0.0-beta.3
```

Comment:
I don't think we need these, these are managed by the bom, so we can skip listing them one by one.

Reply:
Yeap, it only needs to list the overridden artifacts.